optuna/samplers/_tpe/sampler.py

```diff
@@ -146,6 +162,16 @@ def __init__(
         self._rng = np.random.RandomState(seed)
         self._random_sampler = RandomSampler(seed=seed)
 
+        self._multivariate = multivariate
+        self._search_space = IntersectionSearchSpace()
+
+        if multivariate:
+            self._raise_experimental_warning_for_multivariate()
+
+    @experimental("2.1.0", name="`multivariate = True` in TPESampler")
+    def _raise_experimental_warning_for_multivariate(self) -> None:
+        pass
```

Comment:
As discussed in https://github.com/optuna/optuna/pull/1777, we have changed our coding style convention for the experimental argument. Could apply the same change in https://github.com/optuna/optuna/pull/1777?

Reply:
Sure.